README.md

```diff
@@ -1,3 +1,25 @@
 ![](assets/party.gif)
 # Eggy-Bowl
 This is a Egg catcher Game. I designed and developed it. I do represent it with name "Eggy-Bowl". It is a 3-D graphical game using Multiple python libraries.
+
+This Game can be played using arrow keys.
+
+Left arrow key : to move catcher left
+
+Right arrow key : to move catcher right
+
+Built With
+### Python
+### tkinter
+### itertools
+### random
```

Comment:
I'd suggest using an unordered list instead of H3s. Additionally, I'd suggest an H3 for the section heading
(For example:
### Built With
- Python
- tkinter
- itertools
- random
Headings are oftentimes used to navigate text in screen-readers. This greatly impacts the accessibility of the document